### PR TITLE
Pragma table info added

### PR DESCRIPTION
**Pull request by juandent**
As per Eugene's suggestion, i added table_info to pragma_t.
Need to write a test?

### CODE REVIEW

**Comment by fnc12** [comment]
@juandent please change destination branch to `dev` not `master`

**Comment by fnc12** [comment]
@juandent please make changes in `dev` folder not `sqlite_orm.h`

**Comment by fnc12** [comment]
https://github.com/fnc12/sqlite_orm/wiki/Contribution-guide @juandent

File: dev/pragma.h [hunk 1]
```diff
@@ -117,6 +117,40 @@ namespace sqlite_orm {
                 return this->get_pragma<std::vector<std::string>>(oss.str());
             }
 
+            // JDH
+            inline std::vector<sqlite_orm::table_info> table_info(const std::string& tableName) const {
```
Comment (fnc12) [inline dev/pragma.h:121]: `inline` isn't required here

File: dev/pragma.h [hunk 2]
```diff
@@ -117,6 +117,40 @@ namespace sqlite_orm {
                 return this->get_pragma<std::vector<std::string>>(oss.str());
             }
 
+            // JDH
```
[the remark this answers is not in the document]
Comment (juandent) [reply dev/pragma.h:120]: ah ok!!

**Comment by fnc12** [comment]
thank @juandent

**Comment by trueqbit** [comment]
I am surprised that the appveyor unit tests succeeded.
@juandent There's an error in [storage_t<>::schema_status<>()](https://github.com/fnc12/sqlite_orm/blob/c009207a273a3137fff96f48f2be3d5ffcd75a0b/dev/storage.h#L840) - the call to `tImpl.sync_schema()` is missing the 3rd argument.

**Comment by fnc12** [comment]
probably because nobody calls `sync_schema` with empty storage. But I maybe wrong. Anyway looks like a template magic which works when you never call a code which will not compile and the whole app compiles well

**Comment by trueqbit** [comment]
Unit tests don't compile on my end, using Visual C++.

**Comment by fnc12** [comment]
on man. @juandent also uses vs. Stranger things. @juandent did unit tests run well on your machine?

**Comment by trueqbit** [comment]
No surprise here: _tests/sync_schema_tests.cpp_ is missing from _tests/CMakeLists.txt_.

**Pull request merged**